advanced_filters/models.py

```diff
@@ -19,8 +19,12 @@ class Meta:
         verbose_name_plural = _('Advanced Filters')
 
     title = models.CharField(max_length=255, null=False, blank=False, verbose_name=_('Title'))
-    created_by = models.ForeignKey(settings.AUTH_USER_MODEL,
-                                   related_name='created_advanced_filters', verbose_name=_('Created by'))
+    created_by = models.ForeignKey(
+        settings.AUTH_USER_MODEL,
+        related_name='created_advanced_filters',
+        verbose_name=_('Created by'),
+        on_delete=models.CASCADE,
```

Comment:
This is fine right now, but may need to be altered if we implement AdvancedFilter sharing.